ci/pipelines/concourse.yml

```diff
@@ -782,20 +798,174 @@ jobs:
       vars_files:
       - prod/wings/vars.yml
 
-- name: push-docs
+- name: shipit
   public: true
+  serial_groups: [version]
+  plan:
+  - get: concourse
+    passed:
+    - build-rc
+    - k8s-topgun
+    - bosh-smoke
+    - bosh-topgun
+  - get: unit-image
+    passed:
+    - build-rc
+    - k8s-topgun
+    - bosh-smoke
+    - bosh-topgun
+  - get: final-version
+    resource: version
+    params: {bump: final}
+    passed:
+    - build-rc
+    - k8s-topgun
+    - bosh-smoke
+    - bosh-topgun
+  - get: linux-rc
+    passed: [build-rc]
+  - get: windows-rc
+    passed: [build-rc]
+  - get: darwin-rc
+    passed: [build-rc]
+  - get: concourse-rc-image
+    passed: [k8s-topgun]
+  - get: charts
+    passed: [k8s-topgun]
+  - get: concourse-release
+    passed: [bosh-smoke, bosh-topgun]
+  - get: bpm-release
+    passed: [bosh-smoke, bosh-topgun]
+  - get: postgres-release
+    passed: [bosh-smoke, bosh-topgun]
+  - put: version
+    params: {file: final-version/version}
+
+- name: publish-binaries
   serial: true
   plan:
   - aggregate:
+    - get: version
+      passed: [shipit]
+      trigger: true
+    - get: concourse
+      passed: [shipit]
+    - get: unit-image
+      passed: [shipit]
+    - get: linux-rc
+      passed: [shipit]
+    - get: windows-rc
+      passed: [shipit]
+    - get: darwin-rc
+      passed: [shipit]
+  - task: extract-fly-assets
+    file: concourse/ci/tasks/extract-fly-assets.yml
+    image: unit-image
+  - task: build-release-notes
+    file: concourse/ci/tasks/build-release-notes.yml
+    image: unit-image
+  - put: concourse-github-release
+    params:
+      commitish: concourse/.git/ref
+      tag: version/version
+      tag_prefix: v
+      name: release-notes/release-name
+      body: release-notes/notes.md
+      globs:
+      - linux-rc/*.tgz
+      - darwin-rc/*.tgz
+      - windows-rc/*.zip
+      - fly-linux/fly-linux-*.tgz
+      - fly-windows/fly-windows-*.zip
+      - fly-darwin/fly-darwin-*.tgz
+
+- name: publish-bosh-release
+  serial: true
+  plan:
+  - aggregate:
+    - get: version
+      passed: [shipit]
+      trigger: true
+    - get: concourse-release
+      passed: [shipit]
+  - put: concourse-release
+    params:
+      tarball: concourse-release/*.tgz
+      version: version/version
+
+- name: publish-bosh-deployment
+  serial: true
+  plan:
+  - aggregate:
+    - get: concourse-boshio
+      trigger: true
+    - get: unit-image
+      passed: [shipit]
+    - get: cbd
+    - get: version
+      passed: [shipit]
+    - get: bpm-release
+      passed: [shipit]
+    - get: postgres-release
+      passed: [shipit]
+  - task: bump-versions
+    file: cbd/ci/bump-versions.yml
```

Comment:
Do we need to update https://github.com/concourse/concourse-bosh-deployment/blob/f26db87ba8837ef73935fc33f3038dcb0874a363/ci/scripts/bump-versions#L12-L19?

It still references `garden_runc` there 🤔

Reply:
Already updated on `develop`:

https://github.com/concourse/concourse-bosh-deployment/blob/develop/ci/scripts/bump-versions#L12-L19